scrapyd_k8s/k8s_scheduler/__init__.py

```diff
@@ -0,0 +1 @@
+from scrapyd_k8s.k8s_scheduler.k8s_scheduler import KubernetesScheduler
```

Comment:
As to my current understanding, the Kubernetes scheduler is part of the Kubernetes launcher (it has nothing to do with the repository, and it is also not involved in joblogs). So I would expect this to be somehow grouped with the Kubernetes launcher: scheduling is an aspect of launching.

Is that a view that matches yours?
If so, I would expect this to be reflected in the file structure.